packages/blade/src/components/Divider/__tests__/__snapshots__/Divider.web.test.tsx.snap

```diff
@@ -1,196 +1,41 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<Divider /> should render Divider within flex container 1`] = `
-.c0 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-}
```

Comment:
it seems that all the generated CSS got removed from snapshots this should not happen as we deliberately enabled snapshots to include the CSS rules. Maybe check styled-components repo for jest 29 guides

Reply:
Fixed, it seems `jest-styled-components` is not compatible with jest v28+.
- https://github.com/styled-components/jest-styled-components/issues/429